daft/expressions/expressions.py

```diff
@@ -1517,19 +1517,28 @@ def between(self, lower: Any, upper: Any) -> Expression:
         expr = self._expr.between(lower._expr, upper._expr)
         return Expression._from_pyexpr(expr)
 
-    def hash(self, seed: Any | None = None) -> Expression:
+    def hash(
+        self, seed: Any | None = None, hash_function: Literal["xxhash", "murmurhash3", "sha1"] | None = None
+    ) -> Expression:
```

Comment:
The documentation states that it defaults to "xxhash", but the default argument is `None`. Could you please set the default to `= "xxhash"` so this is clear in the code as well? I imagine the plumbing on the rust side is what's mapping None -> xxhash, but this is not clear.